docs/_includes/initialize_shuffle.html

```diff
@@ -1,139 +0,0 @@
-<script type="text/javascript">
-$(document).ready(function() {
```

Comment:
Correct. `js/shuffle.js` will be included by `docs/_layout/default.html` at the end after the common js files via the page variable "additional_js". Since we use shuffle only for the main page, I'm not including it in every page, but only explicitly in `index.md`:

```
additional_js:
 - assets/Shuffle-5.2.3/dist/shuffle.min.js
 - assets/jquery-ui-1.12.1/jquery-ui.min.js
 - js/shuffle.js
```